app/controllers/organization_account_requests_controller.rb

```diff
@@ -0,0 +1,34 @@
+class OrganizationAccountRequestsController < ApplicationController
+  skip_before_action :authenticate_user!
+  skip_verify_authorized
+
+  def new
+    @organization_acccount_request = Organization.new
+  end
+
+  def create
+    @organization_acccount_request = Organization.new(organization_params)
+
+    if @organization_acccount_request.valid?
+      OrganizationAccountRequestsMailer.with(organization_params)
+        .create_new_organization_account_request.deliver_now
+
+      redirect_to root_path, notice: I18n.t("contacts.create.success")
```

Comment:
```suggestion
    if @contact.valid?
      OrganizationAccountRequestMailer.with(contact_params).send_message.deliver_later
      redirect_to root_path, notice: I18n.t("contacts.create.success")
```

Reply:
It might be nice to add a flash message here in the success case

Reply:
Do you mean a different type of flash message than the notice?

Reply:
@Meri-MG Sorry, I completely looked past the fact that you had already added one 🫠